### PR TITLE
WF-995: implement launcher support

### PR DESCRIPTION
**Pull request by JonasImprobable**
We are using `bootstrap.sh` and `bootstrap.bat` instead of starting the client executable directly because it would otherwise silently run in the background. WIth the help of the bootstrap scripts, the clients are run in a new terminal window so that you can see their output and close them more easily.

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 # Changelog
 
+* **23 May 2019:** You can now start client workers for cloud deployments through the [Launcher](https://docs.improbable.io/reference/latest/shared/operate/launcher).
```
Comment (anne-edwards) [inline changelog.md]:
```suggestion
* **23 May 2019:** You can now start client-worker instances for cloud deployments using the [Launcher](https://docs.improbable.io/reference/latest/shared/operate/launcher).
```

File: client/src/Client.cs [hunk 2]
```diff
@@ -33,34 +30,35 @@ static int Main(string[] arguments)
...
-                Console.WriteLine("    <client_id>     - name of the client.");
-                Console.WriteLine("Alternatively: Client <snapshotfile> will generate a snapshot and exit.");
+                Console.WriteLine("Usage 1: Client local <hostname> <port> <client id>");
+                Console.WriteLine("Connects to a local deployment.");
+                Console.WriteLine("");
+                Console.WriteLine("Usage 2: Client cloud <hostname> <player identity token> <login token>");
+                Console.WriteLine("Connects to a cloud deployment.");
+                Console.WriteLine("");
+                Console.WriteLine("Usage 3: Client snapshot <snapshot file>");
+                Console.WriteLine("Generate a snapshot and exit.");
```
Comment (anne-edwards) [inline client/src/Client.cs]: Minor thing (consistency) - should this be "Generates a snapshot and exits" to match the verb forms used in 1 and 2? (Or alternatively we could change the others to "Connect".)

File: changelog.md (hunk 1 above)
Comment (JonasImprobable) [reply changelog.md]: Fixed.

File: client/src/Client.cs (hunk 2 above)
Comment (JonasImprobable) [reply client/src/Client.cs]: Good point. Changed.

**Pull request merged**